packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.jsx

```diff
@@ -460,8 +451,7 @@ const AccountTransferForm = ({
                                             onChangeTransferFrom(e);
                                             handleChange(e);
                                             setFieldValue('amount', '');
-                                            setFieldError('amount', '');
-                                            setFieldTouched('amount', false);
+                                            setTimeout(() => setFieldError('amount', ''));
```

Comment:
we need setTimeout wrapper to remove an error under amount input,  when the user select another fiat account from selected_from dropdown menu